.github/renovate.json

```diff
@@ -170,6 +170,15 @@
         "renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s.* '(?<currentValue>.*)'\\s"
       ],
       "versioningTemplate": "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["package\\.json"],
+      "matchStrings": [
+        "mcr\\.microsoft\\.com\\\/playwright:(?<currentValue>.*?) \\\/bin\\\/bash"
```

Comment:
Should work generally but it might be more robust to change the end from ` \\\/bin\\\/bash` to `\\s` only to avoid relying on the subsequent command string 🤔

Reply:
@cmur2 done